crates/frontend/src/components/context_form/utils.rs

```diff
@@ -186,7 +187,15 @@ pub fn construct_context(
     } else {
         let condition_schemas = conditions
             .iter()
-            .map(|condition| get_condition_schema(condition, dimensions.clone()).unwrap())
+            .map(
```

Comment:
shouldn't this be a filter_map ?
for `Ok(ans)` we can return `Some(ans)`
and for `Err` we can return `None` directly, returning `Null` [here](https://github.com/juspay/superposition/pull/285/files#diff-564227801c44858d95faa622d6194be18193e660848f20a138956b1fd2a15efbR195) seemed wrong

Reply:
I want the parsing to fail with Null, I don't want to filter out failed values and still proceed. I want to fail on wrong inputs